recipes/anaconda-verify/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "anaconda-verify" %}
+{% set version = "1.2.0" %}
+{% set checksum = "e790262f88b3c3ef4ac9d647e0a385ea8ec665ba34509879d0a2270456f5220e" %}
+
+package:
+  name: {{ name.lower() }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ContinuumIO/anaconda-verify/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - anaconda-verify = anaconda_verify.main:main
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - pyyaml
+
+test:
+  imports:
+    - anaconda_verify
+
+  commands:
+    - anaconda-verify --help
+
+about:
+  home: https://github.com/Anaconda-Platform/anaconda-client
+  license: BSD 3-Clause
+  summary: create installer from conda packages
```

Comment:
Ironically this one came from [upstream]( https://github.com/ContinuumIO/anaconda-verify/blob/9b58cb37ef5750faf9be2ecf95fc78e7cc4e7004/setup.py#L21 ), but I agree it should be changed. See this PR ( https://github.com/ContinuumIO/anaconda-verify/pull/5 ). Though I must have forgotten to do that here.